website/content/docs/getting-started.md

```diff
@@ -175,6 +196,59 @@ The included example topologies:
 * `TaskHookTopology.java` --- A topology that uses a task hook to subscribe to
    event notifications.
 
+## Frequently Asked Questions
+
+### 1. How do I know if my topology has been successfully submitted?
+
+- The following message would appear upon successful submission:
+
+```bash
+  INFO: Topology 'ExclamationTopology' launched successfully
+```
+- Note even if the topology is submitted successfully, it could still fail to start some component. For example, `stmgr` may fail to start due to unfulfilled dependencies. To troubleshoot, refer to question 3 and the debugging example below.
+
+### 2. How do I know if my topology is running normally?
+- Heron Tracker and Heron UI provides information about your Heron cluster and topologies. Refer to Step 3 and 4 above.
+
+### 3. How can I debug if my topology failed to start?
+- `~/.herondata/topologies/{cluster}/{role}/{topologyName}/heron-executor.stdout` and `~/.herondata/topologies/{cluster}/{role}/{topologyName}/log-files/` contain helpful information about what went wrong.
+- Installation of `libunwind` is required. Upgrade of `gcc` and `glibc` might be needed.
```

Comment:
compiling.md states minimum versions. let's refer to that to be specific about required gcc and glibc versions.

For libunwind, can we include that in a section that shows what goes wrong without it? Including it in the text here as is makes it hard for someone to realize that that's the cause a given issue that occurs without it.